### DISCUSSION TITLE
繁體中文的Detect Language失效

### DISCUSSION THREAD

**Discussion by hdsbook** [YiiGuxing/TranslationPlugin#5789] [Q&A] [answered]
好像某一版更新之後，輸入繁體中文就會誤偵測為輸入的是英文，希望可以改善，感謝！

![image](https://github.com/YiiGuxing/TranslationPlugin/assets/9463561/57115087-66d9-441e-aa62-9179c3c61e30)

![image](https://github.com/YiiGuxing/TranslationPlugin/assets/9463561/c27f449b-0163-4c89-80ac-2633b4ed120f)

**Comment by YiiGuxing** [comment]
请问你当前使用的是哪个版本的插件，使用的是哪个翻译翻译引擎？

**Reply by hdsbook** [reply]
- IDE：JetBrains Rider 2024.1.2
- Translation版本：3.6.1
- Translation engine: Google Translate
- TTS engine: Google TSS
![image](https://github.com/YiiGuxing/TranslationPlugin/assets/9463561/f9b0989a-2398-4c5e-8470-e87358009149)
![image](https://github.com/YiiGuxing/TranslationPlugin/assets/9463561/0a6b0997-d8ad-4f53-bbad-b17c55fe4454)

翻譯引擎選擇Microsoft Translator的話則是會直接顯示錯誤，無法翻譯

---

目前我先往前下載 3.5.7的版本
使用 google translate(如下圖)，仍無法正確偵測
![image](https://github.com/YiiGuxing/TranslationPlugin/assets/9463561/3f410878-3921-45a1-bd5d-d38a5262935e)

使用 Microsoft Translator，可以正常偵測並翻譯
![image](https://github.com/YiiGuxing/TranslationPlugin/assets/9463561/30d83acb-37b1-4742-bb99-37fdd53f5dd3)

**Reply by YiiGuxing** [reply, marked as answer]
微软翻译的问题在下一个版本修复，过两天就发布。而谷歌翻译的我就无能为力了，它本身就识别为英文，那我也没有办法。。。。
